src/packet_analysis/Manager.cc

```diff
@@ -82,14 +82,14 @@ AnalyzerPtr Manager::GetAnalyzer(const std::string& name) {
 }
 
 bool Manager::EnableAnalyzer(EnumVal* tag) {
-    Component* c = Lookup(tag);
+    Component* c = Lookup(tag, false);
```

Comment:
Similarly, it would be great to allow users to transparently work on all mapped analyzers.